### ISSUE TITLE
Anti-adblock-killer does not work

### ISSUE THREAD

**Issue by Albert456** [reek/anti-adblock-killer#2714]
http://www.elperiodico.cat/ca/
http://www.sport.es/es/

A message tells me, you have activated an advertising blocker.

Another web with problems.
http://www.iptvbin.com/

**Comment by mapx-** [comment]
add all these filters
```
@@||iptvbin.com^$generichide
@@||sport.es^$generichide
@@||elperiodico.cat^$generichide
@@||pagead2.googlesyndication.com^$domain=iptvbin.com
@@.png#$domain=iptvbin.com
```

**Comment by Albert456** [comment]
Thank you. What are the steps?

**Comment by mapx-** [comment]
it's simple, add the filters above in your adblocker (as custom filters).

**Comment by Albert456** [comment]
Thank you. I need a tutorial.

**Comment by mapx-** [comment]
ABP (chrome or firefox):
https://adblockplus.org/forum/viewtopic.php?f=2&t=42013

ubo, options, "my filters" , paste the filters there

**Comment by Albert456** [comment]
Thank you. The only available option Anti-adblock-killer is uBlock Origin and can not customize this type of filters.

**Comment by mapx-** [comment]
No. I said you have to add the filters above as custom filters ! (in "my filters" section)

**Comment by Albert456** [comment]
I needed another extension to accomplish that. Tampermonkey a requirement.
